application-admintools-ui/src/main/resources/AdminTools/Code/AdminToolsJS.xml

```diff
@@ -246,6 +248,37 @@
       $('#downloadFilesModal .download-logs-date-fields').hide();
     }
   });
+
+  // Currently, the Dashboard gadgets title doesn't have an ID set (https://jira.xwiki.org/browse/XWIKI-20600). Solution
+  // is to check for an empty ID and remove the said title as it does not point anywhere in the page. Also, this
+  // function will check for an empty name (TOC macro will generate a title with the name of the page if the
+  // header is empty; this behaviour is modified by https://github.com/xwiki/xwiki-rendering/pull/253/).
```

Comment:
The best practice is to mention issues by ID: title, instead of link. E.g. XWIKI-20600: Gadget headings don't have an ID set

`Remove broken dashboard gadget titles, since in some cases these will be either empty, with the wrong title or will have no ID set, which will make the link unusable. This method should be updated after upgrading to a XWiki parent that includes fixes for XWIKI-20600: Gadget headings don't have an ID set, [ID and name for the issue with empty titles] or XRENDERING-707: The toc macro use the name of the page for empty headers entries.`